cdap-docs/integrations/source/partners/cloudera/ingesting.rst

```diff
@@ -175,12 +175,12 @@ To do this, write the following JSON to a config file::
 the ``schema`` values) to be a conforming JSON file. 
 
 Then use your config file with the ``cdap-etl-batch`` artifact to create an application through the CLI.
-For example, if you wrote the above JSON to a file named ``conversion.json``::
+For example, if you wrote the above JSON to a file named ``conversion.json``:
 
-  .. container:: highlight
+.. container:: highlight
 
-    .. parsed-literal::
-      cdap > create app trades_conversion cdap-etl-batch |version| system <path-to-conversion.json>
+  .. parsed-literal::
+    cdap > create app trades_conversion cdap-etl-batch |release| system <path-to-conversion.json>
```

Comment:
`|release|` is `3.4.0-SNAPSHOT`, while `|version|` is just `3.4.0`. Though it only matters for the SNAPSHOT documentation, it in general is what should be used.